bacardi.cmd

```diff
@@ -1,17 +1,24 @@
-:: Copyright (c) 2017 The Lunch Class Authors.
```

Comment:
I don't think you need to modify this file.
If possible, we should explicitly call the node binary.
For example, we can call as follows:
```<@(PRODUCT_DIR)/../../third_party/node/node.exe path/to/blah/blah/tsc.cmd```

Reply:
i tried this way, but it was needed node_api.h.
```
C:\projects\bacardi\examples/calculator_bridge.h(20): fatal error C1083: Cannot open include file: 'node_api.h': No such file or directory (compiling source file ..\examples\calculator_bridge.cc) [C:\projects\bacardi\build\bacardi.vcxproj]
```